src/Core/Services/Implementations/CollectionService.cs

```diff
@@ -60,7 +60,7 @@ public class CollectionService : ICollectionService
         {
             var groupHasManageAccess = groupsList?.Any(g => g.Manage) ?? false;
             var userHasManageAccess = usersList?.Any(u => u.Manage) ?? false;
-            if (!groupHasManageAccess && !userHasManageAccess)
+            if (!groupHasManageAccess && !userHasManageAccess && !org.AllowAdminAccessToAllCollectionItems)
             {
                 throw new BadRequestException(
                     "At least one member or group must have can manage permission.");
```

Comment:
You also need to update the feature flag check on line 59 to use the v1 flag instead of the MVP flag. (to match the client-side change)

Reply:
We discussed this in the sync, but just closing the loop - we decided it would be safest to leave it in to cover a rollback scenario, where the flag is turned off again after db values have been modified. It also makes life easier for QA who are messing around with their data. (I like that you're thinking critically about flags though, aside from those edge cases I would agree.)